### ISSUE TITLE
Features integration and basic data analysis

### ISSUE THREAD

**Issue by gbatsis** [gbatsis/heart-failure-predictor#3]
### Add routes

Now we have a two-page app: Home page is for predictions and Archive pages will keep the records.

### Features integration
1. Integrate all features and forwarding them to the backend when user presses "Submit" button.
2. Return Python's API response to the frontend.
3. Frontend ensured the validity of feature values (e.g., age between 0 - 10).

### Basic data analysis
Create a notebook (backend/notebooks) with these functionalities:
1. Display dataset records.
2. Check for missing values
3. Check unique values for categorical features and range for numeric
4. Use sklearn's pipeline to combine a data processing workflow and a simple Random forest classifier with default hyperparameters. Data processing methods are: One-hot encoding for categorical features and normalization for numeric.
5. Train classifier and testing.
6. Add these methods to the backend API.
